rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -319,19 +330,18 @@ def output_specs_train(self) -> ModelSpec:
                 SampleBatch.ACTION_LOGP: TorchTensorSpec("b", dtype=torch.float32),
                 SampleBatch.VF_PREDS: TorchTensorSpec("b", dtype=torch.float32),
                 "entropy": TorchTensorSpec("b", dtype=torch.float32),
-                "vf_preds_next_obs": TorchTensorSpec("b", dtype=torch.float32),
```

Comment:
This is not really needed anywhere. I don't know why I originally put it here. But the PPO still learns fine and throughput is also better since we are not doing this computation.